guides/release/upgrading/editions.md

```diff
@@ -91,16 +91,11 @@ Here’s are some of the core features in Octane:
 - **Tracked properties**, a type of decorator that simplifies keeping the DOM
   up-to-date with JavaScript changes.
 - **Async functions** (`async`/`await`) for authoring asynchronous code.
-- **Native JavaScript modules**, including
-  - **Distributing Ember as npm packages**.
-  - **Importing npm packages** with zero additional configuration.
```

Comment:
Did we descope this one? We're still including `ember-auto-import` in the Octane blueprint. I think it's important to find a way to include this, and any remaining issues I think we can get done in the preview period.

Reply:
We didn't descope

Reply:
added it back in